# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/frontend/BytecodeControlStructures.h
# Commit: 7a9ab8423e85
# Full Hash: 7a9ab8423e85a8073f1bcf2ecceb91b46fad6b27
# Author: Jan de Mooij <jdemooij@mozilla.com>
# Date: 2019-12-06 21:48:33
# Regressor Bug: 1601897
# File Overlap Count: 2
# Description:
#   Bug 1601897 part 1 - Remove unused LoopControl::loopEndOffset_. r=arai
#   
#   No longer used since bug 1598548.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D56171
# ==============================================================================

diff -r f3590d2512df -r 7a9ab8423e85 js/src/frontend/BytecodeControlStructures.h
--- a/js/src/frontend/BytecodeControlStructures.h	Fri Dec 06 13:06:21 2019 +0000
+++ b/js/src/frontend/BytecodeControlStructures.h	Fri Dec 06 11:08:02 2019 +0000
@@ -106,9 +106,6 @@
   //
   //   breakTarget:
 
-  // The offset of backward jump at the end of loop.
-  BytecodeOffset loopEndOffset_ = BytecodeOffset::invalidOffset();
-
   // The bytecode offset of JSOP_LOOPHEAD.
   JumpTarget head_;
 
@@ -132,7 +129,6 @@
   LoopControl(BytecodeEmitter* bce, StatementKind loopKind);
 
   BytecodeOffset headOffset() const { return head_.offset; }
-  BytecodeOffset loopEndOffset() const { return loopEndOffset_; }
   BytecodeOffset breakTargetOffset() const { return breakTarget_.offset; }
 
   MOZ_MUST_USE bool emitContinueTarget(BytecodeEmitter* bce);
